admin-dev/themes/new-theme/js/pages/order/view.js

```diff
@@ -37,4 +43,35 @@ $(() => {
       $paymentDetailRow.toggleClass('d-none');
     });
   }
+
+  function togglePrivateNoteBlock() {
+    const $block = $(OrderViewPageMap.privateNoteBlock);
+    const $btn = $(OrderViewPageMap.privateNoteToggleBtn);
+    const isPrivateNoteOpened = $btn.hasClass('is-opened');
+
+    if (isPrivateNoteOpened) {
+      $btn.removeClass('is-opened');
+      $block.addClass('d-none');
+    } else {
+      $btn.addClass('is-opened');
+      $block.removeClass('d-none');
+    }
+
+    const $icon = $btn.find('.material-icons');
+    $icon.text(isPrivateNoteOpened ? 'add' : 'remove');
+  }
+
+  function handlePrivateNoteChange() {
+    const $submitBtn = $(OrderViewPageMap.privateNoteSubmitBtn);
+
+    $(OrderViewPageMap.privateNoteInput).on('input', (event) => {
+      const note = $(event.currentTarget).val();
+
+      if (note) {
+        $submitBtn.attr('disabled', false);
```

Comment:
It's not an attribute, it's a property
```suggestion
        $submitBtn.prop('disabled', !note);
```